okhttp/src/test/java/okhttp3/internal/concurrent/TaskRunnerTest.kt

```diff
@@ -208,10 +193,24 @@ class TaskRunnerTest {
         redQueue.cancelAll()
         return -1L
       }
+    }, 100L)
 
-      override fun tryCancel(): Boolean {
-        log += "cancel@${taskFaker.nanoTime}"
-        return true
+    taskFaker.advanceUntil(0L)
+    assertThat(log).isEmpty()
+
+    taskFaker.advanceUntil(100L)
+    assertThat(log).containsExactly("run@100")
+
+    taskFaker.assertNoMoreTasks()
+  }
+
+  @Test fun cancelWhileExecutingDoesNotStopUncancelableTask() {
```

Comment:
FYI @yschimke this is new since you approved! I added it because clarifying the docs reminded me of a case I’d missed.